cmd/fs/helpers.go

```diff
@@ -46,6 +48,65 @@ func filerForPath(ctx context.Context, fullPath string) (filer.Filer, string, er
 	return f, path, err
 }
 
+const dbfsPrefix string = "dbfs:/"
+
 func isDbfsPath(path string) bool {
-	return strings.HasPrefix(path, "dbfs:/")
+	return strings.HasPrefix(path, dbfsPrefix)
+}
+
+func getValidArgsFunction(pathArgCount int, onlyDirs bool, filerForPathFunc func(ctx context.Context, fullPath string) (filer.Filer, string, error)) func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	return func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		cmd.SetContext(root.SkipPrompt(cmd.Context()))
+
+		err := mustWorkspaceClient(cmd, args) // TODO: Fix this
+		if err != nil {
+			return nil, cobra.ShellCompDirectiveError
+		}
+
+		isValidPrefix := isDbfsPath(toComplete)
+
+		if !isValidPrefix {
```

Comment:
The `fs` commands accept both local and remote paths as args. This is important for the `fs cp` command. 

Could we show the `dbfs:/` prefix as the first option (maybe with highlighting), and the default local paths as alternates?

Reply:
Good point! What do you think about offering both local and dbfs as top level choices when getting started?

<img width="428" alt="Screenshot 2024-07-25 at 10 56 07" src="https://github.com/user-attachments/assets/aad3959c-5def-4a41-990d-7a249fc71efa">

Reply:
`/` might be confusing since autocomplete normally works for local paths. Also for windows, the root path is not `/` but instead `c:/` or `d:/`.

I was thinking something like:
```
dbfs:/ (highlighted / colored so users know it's special, we already use a coloring library in the CLI)
local-file-a
local-file-b
...
```

What do you think about this?

Reply:
Alternately, the autocompletion seems to have a selector functionality, maybe that can be leveraged to decide between local or dbfs:/ paths? I'm not sure though whether it'll work across different shells or on windows.

<img width="1499" alt="Screenshot 2024-07-25 at 11 26 56 AM" src="https://github.com/user-attachments/assets/dc4823e9-ea94-42e7-aa44-9c3e5ca33f76">

Reply:
I made dbfs:/ show up alongside local paths. 

The autocomplete selector functionality that is implemented using shell autocompletion scrhipts enforces alphabetic ordering and doesn't respect the highlighting, I can look into that next:
<img width="545" alt="Screenshot 2024-07-29 at 11 40 41" src="https://github.com/user-attachments/assets/4fdc34c1-cd85-4953-9c77-6c7e0d35a63b">